en/docs/design/prototype-api/overview.md

```diff
@@ -0,0 +1,28 @@
+# Prototyped APIs (Pre-Released APIs)
+
+Prototyped APIs (Pre-Released APIs) allow subscribers to try out APIs without monetizations, allowing them to provide feedback to improve APIs. 
+
+**PRE-RELEASED** (previously **PROTOTYPED**) is a lifecycle state of the API that disallows adding monetizations to the API. After a period of time, the publishers can make changes requested by the users and publish the API (change the lifecycle state of the API to PUBLISHED) to add monetizations as required.
+
+[![PRE-RELEASED lifecycle status]({{base_path}}/assets/img/learn/prototype-api/prototype-api-lifecycle.png)]({{base_path}}/assets/img/learn/prototype-api/prototype-api-lifecycle.png)
+
```

Comment:
Seems image links are not working. Can you check ?
<img width="694" alt="image" src="https://user-images.githubusercontent.com/40932779/161966504-cd7f6ad0-e8e7-48c6-965d-a03af61303a6.png">

Reply:
Can you check again by uncommenting and commenting the two lines here https://github.com/wso2/docs-apim/blob/678ebfb4da7ba945fedfee615b60406cd408afbe/en/mkdocs.yml#L2189